src/probnum/diffeq/odefilter/_odefilter.py

```diff
@@ -145,50 +148,59 @@ def attempt_step(self, state, dt):
 
         It goes as follows:
 
-        1. The current state :math:`x(t) \sim \mathcal{N}(m(t), P(t))` is split into a deterministic component
-        and a noisy component,
+        1. The current state :math:`x(t) \sim \mathcal{N}(m(t), P(t))` is split into a
+        deterministic component and a noisy component,
 
         .. math::
             x(t) = m(t) + p(t), \quad p(t) \sim \mathcal{N}(0, P(t))
 
-        which is required for accurate calibration and error estimation: in order to only work with the local error,
-        ODE solvers often assume that the error at the previous step is zero.
+        which is required for accurate calibration and error estimation: in order to
+        only work with the local error, ODE solvers often assume that the error at the
+        previous step is zero.
 
-        2. The deterministic component is propagated through dynamics model and measurement model
+        2. The deterministic component is propagated through dynamics model and
+        measurement model
 
         .. math::
-            \hat{z}(t + \Delta t) \sim \mathcal{N}(H \Phi(\Delta t) m(t), H Q(\Delta t) H^\top)
+        \hat{z}(t+\Delta t)\sim\mathcal{N}(H \Phi(\Delta t) m(t), H Q(\Delta t) H^\top)
 
-        which is a random variable that estimates the expected local defect :math:`\dot y - f(y)`.
+        which is a random variable that estimates the expected local defect
+        :math:`\dot y - f(y)`.
 
-        3. The counterpart of :math:`\hat{z}` in the (much more likely) interpretation of an erronous previous state
-        (recall that :math:`\hat z` was based on the interpretation of an error-free previous state) is computed as,
+        3. The counterpart of :math:`\hat{z}` in the (much more likely) interpretation
+        of an erronous previous state (recall that :math:`\hat z` was based on the
+        interpretation of an error-free previous state) is computed as,
 
         .. math::
-            z(t + \Delta t) \sim \mathcal{N}(\mathbb{E}[\hat{z}(t + \Delta t)],
-            \mathbb{C}[\hat{z}(t + \Delta t)] + H \Phi(\Delta t) P(t) \Phi(\Delta t)^\top H^\top ),
+        z(t + \Delta t) \sim \mathcal{N}(\mathbb{E}[\hat{z}(t + \Delta t)],
+        \mathbb{C}[\hat{z}(t+\Delta t)]+H\Phi(\Delta t)P(t)\Phi(\Delta t)^\topH^\top),
```

Comment:
Could the `\topH` be the reason for the failed doc-build? It seems to me that this is not valid latex.

Reply:
good catch, i compiled the latex code in Overleaf and realized the spacing would augment the formula